appsdk/sdk.go

```diff
@@ -90,6 +93,7 @@ type AppFunctionsSDK struct {
 	configProfile             string
 	configDir                 string
 	useRegistry               bool
+	ignoreVersionError        bool
```

Comment:
I'm not in love with this being called  ignoreVersionError...what about ignoreVersionCheck?

Reply:
Code still does version check and logs error, but if ignoreVersionError is set it does't exit.